frontend/src/pages/Confirmation/ConfirmationDialog.js

```diff
@@ -497,9 +501,20 @@ const ConfirmationDialog = props => {
 
         title = strings.confirmation.subproject_close;
         content = <Typography>{dialogText}</Typography>;
-        confirmButtonText = strings.confirmation.subproject_close;
+        confirmButtonText = strings.confirmation.workflowitem_close_accept;
         break;
       }
+      case "workflowitem.reject": {
+        const dialogText = {
+          commentLabel:strings.common.comment,
+          commentPlaceholder:strings.common.comment_description
+        };
```

Comment:
As I mentioned in sagas.js, remove this since this intent does not exist. Instead, update "workflowitem.close"